### ISSUE TITLE
Cannot meta-npm command to work on windows 10

### ISSUE THREAD

**Issue by wendellmva** [mateodelnorte/meta-npm#44]
## 🐛 Bug Report

Trying to use meta-npm gives this error before installing it (assumes it comes with meta like help suggests:

> meta npm install standard-version --save=dev

>Error: could not find meta-npm.cmd
    at findModuleBin (C:\ProgramData\nvm\v13.8.0\node_modules\meta\node_modules\find-module-bin\lib\index.js:18:9)
    at Command.executeSubCommand (C:\ProgramData\nvm\v13.8.0\node_modules\meta\node_modules\commander\index.js:552:11)
    at Command.parse (C:\ProgramData\nvm\v13.8.0\node_modules\meta\node_modules\commander\index.js:489:17)
    at Object.exports.run (C:\ProgramData\nvm\v13.8.0\node_modules\meta\index.js:47:11)
    at Object.<anonymous> (C:\ProgramData\nvm\v13.8.0\node_modules\meta\bin\meta:8:15)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)

Trying to install gives this error:

>npm install meta-npm --save=dev     
                                                                                                                                                                                          
> npm WARN deprecated mkdirp@0.3.5: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for inotify@1.4.6: wanted {"os":"linux","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm ERR! notsup Valid OS:    linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

## To Reproduce

Just try to install it on a windows 10 machine

## Expected behavior

> meta npm install standard-version --save=dev

Expect for all projects their packages.json to contain the latest version of standard-version package in their devDependecies sections.

**Comment by mateodelnorte** [comment]
Mind looking into mkdirp and seeing if the issue is solved in a newer version?

**Comment by wendellmva** [comment]
It works fine, it just doesn't want to install it through npm. I cloned meta-npm git repo directly into the node_modules folder then it started to work as expected. I love the tool. I replaced lerna with meta since its submodules support is lacking. Meta in combo with just-build makes one hell of a power couple. The only thing I wished i could do isa filter something like 
> meta project src/apps/* npm run build

or

> meta project src/apps/app src/backend -c npm run build

I'll look into the above and creating a meta-project plugin to apply a filter to the projects before running the commands.
